doozer/doozerlib/backend/konflux_fbc.py

```
@@ -1147,6 +1151,64 @@ def _extract_git_commits_from_nvrs(self, bundle_nvrs: str) -> list[str]:
+            for image_pullspec in related_images:
+                share_pullspec = image_pullspec.replace(KONFLUX_DEFAULT_IMAGE_REPO, KONFLUX_ART_IMAGES_SHARE)
+
+                if await self._check_image_exists(share_pullspec, logger):
+                    logger.info(f"Image already exists in art-images-share: {share_pullspec}")
+                else:
+                    logger.info(f"Syncing {image_pullspec} to art-images-share")
+                    try:
+                        destination_repo = share_pullspec.split('@')[0]  # Remove digest for destination
+                        await artlib_util.sync_to_quay(image_pullspec, destination_repo)
+                        logger.info(f"Successfully synced to art-images-share: {share_pullspec}")
+                    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)